curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/655220953ba90d80514d7ee2.md

```diff
@@ -1,28 +1,27 @@
 ---
 id: 655220953ba90d80514d7ee2
-title: Step 24
+title: Step 28
 challengeType: 20
-dashedName: step-24
+dashedName: step-28
 ---
 
 # --description--
 
-An *argument* is an object or an expression passed to a function — between the parentheses — when it is called. The `print` function can take multiple arguments, separated by a comma.
+An *argument* is an object or an expression passed to a function — included between the parentheses — when it is called. The `print` function can take multiple arguments, separated by a comma.
```

Comment:
It is better to move the description of an argument to step 4 where it is first introduced.